week1/responses/sendIndexHTML.js

```diff
@@ -0,0 +1,6 @@
+export default function sendIndexHTML (response,state,calculation){
```

Comment:
You could simplify this function to only send the state when a state is passed. For example:

```javascript
export default function sendIndexHTML (response, state) {
  response.setHeader('Content-type','text/html')
  response.write(
    `<html><header><title>calculator</title></header><body>${state || 'ok'}</body></html>`
  )
}
```